config/site_settings.yml

```diff
@@ -3549,3 +3549,7 @@ experimental:
   use_overhauled_theme_color_palette:
     default: false
     hidden: true
+  experimental_rich_editor:
```

Comment:
Having `experimental_` here means we're gonna have to eventually rename it and write a database migration.

How about just calling it 'rich_editor', and making sure that the risks are described in the the description (when we eventually unhide the setting).